<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceStatusWindow</class>
 <widget class="QFrame" name="DeviceStatusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeviceTestWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	background-color: #f2f1ef;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDeviceType">
     <property name="text">
      <string>#device_type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblDeviceModel">
     <property name="text">
      <string>#device_model</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblDeviceStatus">
     <property name="text">
      <string>#status_undefined</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnRunTest">
     <property name="minimumSize">
      <size>
       <width>86</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>#test</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
